stl/inc/deque

```diff
@@ -832,6 +832,8 @@ public:
 #if _HAS_CXX23
     template <_Container_compatible_range<_Ty> _Rng>
     void assign_range(_Rng&& _Range) {
+        static_assert(_STD assignable_from<_Ty&, _RANGES range_reference_t<_Rng>>,
+            "Elements must be assignable from the range's reference type. N4993 [sequence.reqmts]/60");

```

Comment:
This `_STD`-qualification for a concept is unconventional. We're fastidious about qualifying calls to "pretty" functions (functions whose names are `_Ugly` or `__ugly`) and trying to qualify calls even to `_Ugly` functions in new code to avoid ADL problems, but types and concepts and other not-function-things don't have that problem. (Several occurrences)

We also typically parenthesize citations or call them out in the prose with per (Several occurrences as well):
```suggestion
        static_assert(assignable_from<_Ty&, _RANGES range_reference_t<_Rng>>,
            "Elements must be assignable from the range's reference type (N4993 [sequence.reqmts]/60).");
```